### ISSUE TITLE
Unable to login using Visual Git if you download the application zip from Github

### ISSUE THREAD

**Issue by karthiksriramuoa** [kblincoe/VisualGit_SE701_3#140]
![visualgiterror](https://user-images.githubusercontent.com/31844757/38010326-07c0a584-32b5-11e8-8c04-9653c616b0a3.jpg)
Scenario- 
1. Download the VisualGit Code Repository from Github
2. cd Dir -> npm install
3. npm start.

Expected Behavior -
1. Run this application and login to the application.
2. After you enter your credentials and hit login, you must be able to login and enter the homepage.

Actual Behavior -
1. After running this application and entering the credentials, hit Sign In.
2. The page is frozen and does not show any message. 

Proposed Solution -
-Investigate the authenticate module and check what happens when clicked on Sign In.
-Resolve the issue by authenticating and validating the user against Github credentials.

![visualgitloginerror](https://user-images.githubusercontent.com/31844757/38010730-dd510b8e-32b6-11e8-901b-d9d96b6ac122.jpg)

**Comment by softeng-701** [comment]
@karthiksriramuoa Approved

**Comment by cielluo** [comment]
In fact, after I merged the current upstream repository into my fork, I cannot login, either.
I tried to input a wrong password, and the application demonstrated an error message: "Error: Bad credentials". So I think the sign in function is working, but there are some issues with the login process.

Also, I have an copy of the original zip file of VisualGit as backup. I unzipped and installed it and tried to login in that version, and I succeed.

Maybe the content of the issue should be modified. Some issues appeared after students send PRs.
@softeng-701 @karthiksriramuoa

**Comment by softeng-701** [comment]
@karthiksriramuoa @ls-morning Yes, I think this is a general problem with signing in no matter how you get the code. This issue should look at signing in in general

**Comment by karthiksriramuoa** [comment]
@softeng-701 @ls-morning I reported the sign in issue and an issue with starting the application, to Kelly, but she replied me that it was a problem with the npm version. I am not sure if its because of the same.

**Comment by softeng-701** [comment]
@karthiksriramuoa Did you get it working by changing the version?

**Comment by karthiksriramuoa** [comment]
@softeng-701 No. Changing the version didnt work. So i thought there was an issue with my machine. I hence worked on a cloned repository shared by my teammate.
